airbyte-ci/connectors/pipelines/pipelines/bases.py

```diff
@@ -477,6 +478,9 @@ def print(self):
             failures_group = Group(*sub_panels)
             to_render.append(failures_group)
 
+        if self.pipeline_context.dagger_cloud_url:
+            self.pipeline_context.logger.info(f"🔗 View commit in Dagger Cloud: {self.pipeline_context.dagger_cloud_url}")
```

Comment:
```suggestion
            self.pipeline_context.logger.info(f"🔗 View runs for commit in Dagger Cloud: {self.pipeline_context.dagger_cloud_url}")
```

Reply:
done in [9ab0f68](https://github.com/airbytehq/airbyte/pull/28947/commits/9ab0f680bea0d851a62a4c2a0bfd530d222dd110)